tests/unittest/_torch/ray_orchestrator/multi_gpu/test_ops.py

```diff
@@ -32,9 +39,8 @@ def __init__(self, rank, world_size):
 
     @torch.inference_mode()
     def run_allgather_pg_op(self, test_tensor, expected_result, sizes):
-        torch.cuda.set_device(0)
-        test_tensor = test_tensor.cuda(0)
-        expected_result = expected_result.cuda(0)
+        test_tensor = test_tensor.to(self.local_device)
```

Comment:
Simply `.cuda()` should move to the current CUDA device. This verifies `torch.cuda.set_device(self.local_gpu)` work properly.